archinstall/lib/hardware.py

```diff
@@ -37,18 +37,22 @@ def get_ucode(self) -> Optional[Path]:
 
 
 class GfxPackage(Enum):
+	Dkms = 'dkms'
 	IntelMediaDriver = 'intel-media-driver'
 	LibvaIntelDriver = 'libva-intel-driver'
 	LibvaMesaDriver = 'libva-mesa-driver'
 	Mesa = "mesa"
-	Nvidia = 'nvidia'
```

Comment:
Won't this make it so `nvidia` package is not installed? Which is needed for some platforms? Or we rely solely on the `-dkms` and add the nvidia hooks to build the driver each kernel upgrade?

Reply:
Maybe I missed something but in the old code Nvidia didn't seem to be installed at all. But please double check that I didn't miss anything

I've actually tried installing nvidia and nvidia-dkms which threw a conflict error